next.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  images: {
+    domains: process.env.NEXT_PUBLIC_IMAGE_DOMAINS.split(", "),
```

Comment:
```suggestion
    domains: process.env.NEXT_PUBLIC_IMAGE_DOMAINS?.split(",")?.map((d)=>d.trim()),
```
What's with the space (" ")?